doc/whatsnew/2.12.rst

```diff
@@ -56,6 +56,11 @@ New checkers
 
   Closes #5208
 
+* A new ``bad-configuration-option-value`` checker was added that will emit for misplaced option
+  in pylint's top level namespace for toml configuration. Top-level dictionaries or option defined
+  in the wrong section will still silently not be taken into account, which is tracked in a
+  follow-up issue.
```

Comment:
Dont't we want to link to the follow up issue here?

Reply:
0fbaadcd